RockWeb/Themes/NewSpring/assets/Lava/LAYOUT/sermon-detail.lava

```diff
@@ -1,75 +1,62 @@
+{% comment %}
+    Redirect if item is expired
+{% endcomment %}
 {% assign pageId = 'Global' | Page:'Id' %}
 {% assign CurrentPersonCanEdit = pageId | HasRightsTo:'Edit','Rock.Model.Page' %}
```

Comment:
If this was removed in `editorial-detail`, does it need to stay in this one?

Reply:
I'm not sure what you're referencing here. We need the redirect to 404 if the item isn't found or is expired.

Reply:
It's not the green highlighted lines. It's like 5 with the `CurrentPersonCanEdit` variable.

Reply:
That's there because we don't redirect if it's expired for content editors - that way they can preview content on the page before it goes live.

Reply:
They don't need that on `editorial-detail`?

Reply:
Again, just want to make sure this is correct. You removed the `CurrentPersonCanEdit` in a few other places so it seems odd that you didn't remove it here. Just checking to make sure.